app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -20,6 +20,8 @@ def ems_inv_to_hashes
     get_vdcs
     get_vapps
     get_vms
+    get_private_images if @options.get_private_images
```

Comment:
Image might be a bad choice of name since neither MIQ nor the provider call it an image.
In this case a vCloud catalog item is going to map to a MIQ template which is a subclass of `VmOrTemplate`, this means that a catalog item should be fairly similar to a vm (i.e. similar to VMware vSphere)

If a catalog item isn't similar to a VM then it should be parsed as something other than a `Template`